bluez-generated/src/leadvertisingmanager1.rs

```diff
@@ -62,3 +62,28 @@ impl<'a, T: nonblock::NonblockReply, C: ::std::ops::Deref<Target = T>> OrgBluezL
         )
     }
 }
+
+pub const ORG_BLUEZ_LEADVERTISING_MANAGER1_NAME: &str = "org.bluez.LEAdvertisingManager1";
+
+#[derive(Copy, Clone, Debug)]
+pub struct OrgBluezLEAdvertisingManager1Properties<'a>(pub &'a arg::PropMap);
+
+impl<'a> OrgBluezLEAdvertisingManager1Properties<'a> {
+    pub fn from_interfaces(
+        interfaces: &'a ::std::collections::HashMap<String, arg::PropMap>,
+    ) -> Option<Self> {
+        interfaces.get("org.bluez.LEAdvertisingManager1").map(Self)
+    }
+
+    pub fn active_instances(&self) -> Option<u8> {
+        arg::prop_cast(self.0, "ActiveInstances").copied()
+    }
+
+    pub fn supported_instances(&self) -> Option<u8> {
+        arg::prop_cast(self.0, "SupportedInstances").copied()
+    }
+
+    pub fn supported_includes(&self) -> Option<&Vec<String>> {
+        arg::prop_cast(self.0, "SupportedIncludes")
+    }
+}
```

Comment:
I feel like this PR validates our earlier decision to make a bluez-generated crate that *only* contains generated code.

Reply:
Generating it as part of a `build.rs` would also work, but this is simpler in some ways.